Tests/EventDispatcherTests.swift

```diff
@@ -0,0 +1,198 @@
+import Quick
+import Nimble
+import Clappr
+
+class EventDispatcherTests: QuickSpec {
+
+    override func spec() {
+        describe("BaseObject") {
```

Comment:
We're testing `BaseObject` or `EventDispatcher`? In the code `BaseObject` uses `EventDispatcher` but they're not the same thing and the code should not have any knowledge about this connection.